src/format/opamPath.ml

```diff
@@ -102,6 +102,9 @@ module Switch = struct
 
   let build t a nv = build_dir t a / OpamPackage.to_string nv
 
+  let build_opam ~build_dir =
+    OpamFilename.of_string ( OpamFilename.Dir.to_string build_dir ^ ".opam" )
```

Comment:
```suggestion
  let build_opam t a nv = build_dir t a /- OpamPackage.to_string nv ^ ".opam"
```

Reply:
That would create the file within the packages sources, no ? I think it's a bit dangerous since it can badly interact with the build/install process (for example, `opam-bin` installs everything that is in the source tree of its binary packages, so it would install the opam file too).